changelogs/unreleased/protobuf_encoder_module.md

```diff
@@ -0,0 +1,3 @@
+## feature/lua
```

Comment:
Nitpicks regarding the commit message/the pull request description.

> Introduce protobuf encoder.

There should be no period at end of a commit title. See the [style guide](https://www.tarantool.io/en/doc/latest/dev_guide/developer_guidelines/#how-to-write-a-commit-message), bullet 4.

> reqursive
> reqursion

q -> c